xscen/ensembles.py

```diff
@@ -1,24 +1,28 @@
 import logging
 from pathlib import Path
-from typing import Optional, Union
 
 import pandas as pd
 import xarray as xr
 from xclim import ensembles
 
-from .catalog import ProjectCatalog, generate_id
+from .catalog import generate_id  # ProjectCatalog
+
+# from typing import Optional, Union
+
 
 logger = logging.getLogger(__name__)
 
+__all__ = ["ensemble_stats"]
+
 
 def ensemble_stats(
     datasets: list,
-    create_kwargs: dict = {},
```

Comment:
why is setting {} as default not okay?

Reply:
https://towardsdatascience.com/python-pitfall-mutable-default-arguments-9385e8265422

Reply:
Oh interesting, I didn't know ! Thank you! I like the principle of least surprise haha